tests/unit/include/es-test.hpp

```diff
@@ -22,8 +22,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 // We require SEH for Windows builds, so we test with SEH support too.
 #define CATCH_CONFIG_WINDOWS_SEH
 // Check for memory leaks from test code.
-// TODO: enable after patching upstream issue
-// #define CATCH_CONFIG_WINDOWS_CRTDBG
+#define CATCH_CONFIG_WINDOWS_CRTDBG
```

Comment:
is this from c++20 or something else?

Reply:
I think that's just from the catch update.

Reply:
so... not for this PR?

Reply:
I wanted to put it in the C++20 pr, just forgot.